psydac/feec/multipatch/fem_linear_operators.py

```diff
@@ -59,6 +59,12 @@ def T(self):
     def dtype( self ):
         return self.domain.dtype
 
+    def toarray(self):
+        raise NotImplementedError('toarray() is not defined for FEMLinearOperators.')
+
+    def tosparse(self):
+        raise NotImplementedError('tosparse() is not defined for FEMLinearOperators.')
```

Comment:
Could we not implement these functions as follows?
```suggestion
    def toarray(self):
        self.matrix.toarray()

    def tosparse(self):
        self.matrix.tosparse()
```
CC: @campospinto

Reply:
True, that should work, will update before the next push!